packages/lit-html/src/lit-html.ts

```diff
@@ -310,7 +310,7 @@ export interface RenderOptions {
    * An object to use as the `this` value for event listeners. It's often
    * useful to set this to the host component rendering a template.
    */
-  host?: object;
+  host?: unknown;
```

Comment:
another one to double check since i guess if we have consumers of `host`, it would be a breaking change?